python/cudnn/__init__.py

```
@@ -271,10 +271,12 @@ def _dlopen_cudnn():
 from .nodes import Node
 
 from .graph import graph, jit, graph_cache
-from .wrapper import Graph
 
 from typing import Any
 
+_EAGER_PUBLIC_NAMES = tuple(name for name in globals() if not name.startswith("_"))
+__all__ = [*_EAGER_PUBLIC_NAMES, "Graph", "wrapper"]
```

Ruff (0.15.21)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)